manuscript/portal_iterative_forecasting_ms.Rmd

```diff
@@ -41,32 +41,32 @@ Key-words: forecasting, prediction, mammals, iterative forecasting, Portal Proje
 
-Iterative forecasting is most effective with frequently collected data, since it provides more opportunities for updating model results and assessing (and potentially improving) model performance [Box 1; @dietze2018]. The Portal Project is a long-term ecological study situated in the Chihuahuan Desert (2 km north and 6.5 km east of Portal, Arizona, US). Researchers have been continuously collecting data at the site since 1977, including data on the abundance of rodent and plant species (monthly and twice yearly, respectively) and climatic factors such as air temperature and precipitation (daily) [@brown1998; @ernest2009; @ernest2016; @ernest2018]. The site consists of 24 50m x 50m experimental plots. Each plot contains 49 permanently marked trapping stations laid out in a 7 x 7 grid, and all plots are trapped with Sherman live traps for one night each month. For all rodents caught during a trapping session, information on species identity, size, and reproductive condition is collected, and new individuals are given identification tags. This information on rodent populations is high-frequency, uses consistent trapping methodology, and has an extended time-series (475 monthly samples and counting), making this study an ideal case for near-term iterative forecasting. 
+Iterative forecasting is most effective with frequently collected data, since it provides more opportunities for updating model results and assessing (and potentially improving) model performance [Box 1; @dietze2018]. The Portal Project is a long-term ecological study situated in the Chihuahuan Desert (2 km north and 6.5 km east of Portal, Arizona, US). Researchers have been continuously collecting data at the site since 1977, including data on the abundance of rodent and plant species (monthly and twice yearly, respectively) and climatic factors such as air temperature and precipitation (daily) [@brown1998; @ernest2009; @ernest2016; @ernest2018]. The site consists of 24 50m x 50m experimental plots. Each plot contains 49 permanently marked trapping stations laid out in a 7 x 7 grid, and all plots are trapped with Sherman live traps for one night each month. For all rodents caught during a trapping session, information on species identity, size, and reproductive condition is collected, and new individuals are given identification tags. This information on rodent populations is high-frequency, uses consistent trapping methodology, and has an extended time-series (475 monthly samples and counting), making this study an ideal case for near-term iterative forecasting. Forecasting of rodent population dynamics in the southwest (and more broadly) is important because of their link to zoonotic diseases such as hantavirus and plague [@parmenter1993; @gage2005; @springer2016]. In addition, this forecasting system is being used to improve population dynamic modeling for this system and to explore the utility of incorporating experimental data into ecological forecasting models. 
```

Comment:
similar to earlier comment re "systems", at the end of this paragraph, I would suggest changing "modeling for this system" to "modeling for this natural system" to continue the distinction between the two "systems".